usr/share/rear/conf/default.conf

```diff
@@ -2925,12 +2925,22 @@ SIMPLIFY_TEAMING=no
 # This config value may change in the near future and may merge with SERIAL_CONSOLE_DEVICE_SYSLINUX.
 USE_SERIAL_CONSOLE=
 
-# Since syslinux does not work with more then one serial device at once it must be explicitly named.
+# Since you may not want to use all serial devices found as console you may configure the ones to use here.
+# You would configure a list of devices to use e.g. 'SERIAL_CONSOLE_DEVICES="/dev/ttyS0 /dev/ttyS1"'.
+# By default (when unset) all serial devices found are used.
+SERIAL_CONSOLE_DEVICES=
+
+# Since syslinux does not always work with more then one serial device at once it can be explicitly named.
 # Here you can configute the serial console device to use e.g. 'SERIAL_CONSOLE_DEVICE_SYSLINUX=/dev/ttyS0' when USE_SERIAL_CONSOLE is turned on.
 # By default (when unset) all found serial devices are used.
 # This config value may change in the near future and may get merged with USE_SERIAL_CONSOLE.
 SERIAL_CONSOLE_DEVICE_SYSLINUX=
```

Comment:
Is there a reason why SERIAL_CONSOLE_DEVICE_SYSLINUX is singular
i.e. why there can be only one SERIAL_CONSOLE_DEVICE_SYSLINUX?